cx-content-moderation/main.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"cloud.google.com/go/compute/metadata"
+	documentai "cloud.google.com/go/documentai/apiv1beta3"
+	"cloud.google.com/go/documentai/apiv1beta3/documentaipb"
+
+	"google.golang.org/api/option"
+
+	"github.com/googlecloudplatform/ezcx"
+)
+
+var (
+	projectID                      string
+	logname                        string
+	contentModerationProcessorName string
+	/*
+		perspectiveKey                 string
+		language                       string
+		languageAttributes             map[string][]string
+	*/
+)
+
+const location = "us"
+
+func main() {
+	log.Printf("cx-content-moderation")
+	// guards for config
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = "8080"
+	}
```

Comment:
Could we simplify this to

```
port := os.Getenv("PORT") || "8080"
```
Not sure if go works this way like python?

Reply:
Go's verbosity is legendary :) There's no logical or construct like this, but I will find a more idiomatic way to do this